artifactory/commands/buildinfo/publish_test.go

```diff
@@ -67,3 +72,34 @@ func TestExcludePasswordsPattern(t *testing.T) {
 		t.Error("expected:", expected, "got:", filteredKeys)
 	}
 }
+
+func TestPrintBuildInfoLink(t *testing.T) {
+	var linkTypes = []struct {
+		artVersion    int
+		buildTime     time.Time
+		buildInfoConf artifactoryUtils.BuildConfiguration
+		serverDetails config.ServerDetails
+		expected      string
+	}{
+		{5, time.Now(), artifactoryUtils.BuildConfiguration{BuildName: "test", BuildNumber: "1", Module: "6", Project: "cli"},
+			config.ServerDetails{Url: "http://localhost:8081/"}, "http://localhost:8081/artifactory/webapp/#/builds/test/1"},
+		{6, time.Now(), artifactoryUtils.BuildConfiguration{BuildName: "test", BuildNumber: "1", Module: "6", Project: "cli"},
+			config.ServerDetails{Url: "http://localhost:8081/"}, "http://localhost:8081/artifactory/webapp/#/builds/test/1"},
+		{7, time.Now(), artifactoryUtils.BuildConfiguration{BuildName: "test", BuildNumber: "1", Module: "6", Project: ""},
+			config.ServerDetails{Url: "http://localhost:8082/"}, "http://localhost:8082/ui/builds/test/1/" + strconv.FormatInt(time.Now().UnixNano()/1000000, 10) + "/published?buildRepo=artifactory-build-info"},
```

Comment:
```strconv.FormatInt(time.Now().UnixNano()/1000000, 10)``` should calculated once at the beginning of the test. The code will simply be shorter.